docs: add note about IPV4 only for APIM frontend deployments (#781)
gravitee-io-labs/beta-docs: docs/apim/4.8/self-hosted-installation-guides/kubernetes/vanilla-kubernetes.md

    # Ensure you have PostgreSQL, Redis, and Elasticsearch services running in your cluster
    ```

> > If your Kubernetes cluster does not support IPV6 networking, both the ui and portal deployments need to define the `IPV4_ONLY` environment variable with `"true"` as a value.

Reviewer: ```suggestion
> If your Kubernetes cluster does not support IPV6 networking, both the UI and Portal deployments must set the `IPV4_ONLY` environment variable to `true`.
```
